src/build-data/policy/bsi.txt

```diff
@@ -153,8 +164,10 @@ sm2
 # pk_pad
 #eme_pkcs1 // needed for tls
 #emsa_pkcs1 // needed for tls
+eme_raw
 emsa_raw
 emsa_x931
+raw_hash
```

Comment:
That's a module to implement certain special cases using Botan. Shouldn't be enabled by default but also not prohibited.

Reply:
This new module is from my understanding now used in e.g. ECDSA there before `emsa_raw` was used (as they required EMSA1(hash) before instead of the hash directly).

Don't know why we currently forbid `emsa_raw`, but we may want to keep it consistent an either forbid both or none.

Reply:
I'll remove it for now.